content/en/docs/tasks/debug/debug-cluster/_index.md

```diff
@@ -0,0 +1,6 @@
+---
+title: "Debugging Clusters"
```

Comment:
How does this section title align with the title of `content/en/docs/tasks/debug/debug-cluster/debug-cluster.md`: “Troubleshooting Clusters”.

I'm thinking that Troubleshooting should be the generic title (not Debugging).